.github/actions/godot-deps/action.yml

```diff
@@ -1,27 +1,30 @@
-name: Setup python and scons
-description: Setup python, install the pip version of scons.
+name: Setup Python and SCons
+description: Setup Python, install the pip version of SCons.
 inputs:
   python-version:
-    description: The python version to use.
+    description: The Python version to use.
     default: "3.x"
   python-arch:
-    description: The python architecture.
+    description: The Python architecture.
     default: "x64"
+  scons-version:
+    description: The SCons version to use.
+    default: "4.7.0"
 runs:
   using: "composite"
   steps:
-    # Use python 3.x release (works cross platform)
     - name: Set up Python 3.x
       uses: actions/setup-python@v5
       with:
-        # Semantic version range syntax or exact version of a Python version
+        # Semantic version range syntax or exact version of a Python version.
         python-version: ${{ inputs.python-version }}
-        # Optional - x64 or x86 architecture, defaults to x64
+        # Optional - x64 or x86 architecture, defaults to x64.
         architecture: ${{ inputs.python-arch }}
 
-    - name: Setup scons
+    - name: Setup SCons
       shell: bash
       run: |
         python -c "import sys; print(sys.version)"
-        python -m pip install scons==4.7.0
+        python -m pip install wheel
```

Comment:
Had to install `wheel` before `scons`, otherwise it failed with this error:
```
3.6.15 (default, Sep  6 2021, 07:07:23) 
[GCC 9.3.0]
Collecting scons==3.0.0
  Downloading scons-3.0.0.tar.gz (630 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Using legacy 'setup.py install' for scons, since package 'wheel' is not installed.
Installing collected packages: scons
    Running setup.py install for scons: started
    Running setup.py install for scons: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /opt/hostedtoolcache/Python/3.6.15/x64/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-c84v8i3g/scons_e21b17358cca4fa0a6913eec3e36b727/setup.py'"'"'; __file__='"'"'/tmp/pip-install-c84v8i3g/scons_e21b17358cca4fa0a6913eec3e36b727/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-9y39lo61/install-record.txt --single-version-externally-managed --compile --install-headers /opt/hostedtoolcache/Python/3.6.15/x64/include/python3.6m/scons
         cwd: /tmp/pip-install-c84v8i3g/scons_e21b17358cca4fa0a6913eec3e36b727/
    Complete output (6 lines):
    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
       or: setup.py --help [cmd1 cmd2 ...]
       or: setup.py --help-commands
       or: setup.py cmd --help
    
    error: option --single-version-externally-managed not recognized
    ----------------------------------------
ERROR: Command errored out with exit status 1: /opt/hostedtoolcache/Python/3.6.15/x64/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-c84v8i3g/scons_e21b17358cca4fa0a6913eec3e36b727/setup.py'"'"'; __file__='"'"'/tmp/pip-install-c84v8i3g/scons_e21b17358cca4fa0a6913eec3e36b727/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-9y39lo61/install-record.txt --single-version-externally-managed --compile --install-headers /opt/hostedtoolcache/Python/3.6.15/x64/include/python3.6m/scons Check the logs for full command output.
```